lenses/nginx.aug

```diff
@@ -33,28 +33,43 @@ autoload xfm
 
 (* Variable: block_re
      The keywords reserved for block entries *)
-let block_re = "http" | "events"
-             | "server" | "upstream"
-             | "location" | "if"
+let block_re = "http" | "events" | "server" | "upstream"
+
```

Comment:
Thanks for pointing this out.

I've gone through the nginx docs now and added the following block types: mail, stream, geo, map, and split_clients

I *hope* that's all of them. If not, please let me know.